app/src/main/java/org/dhis2/usescases/enrollment/EnrollmentActivity.kt

```diff
@@ -499,8 +499,9 @@ class EnrollmentActivity : ActivityGlobalAbstract(), EnrollmentView {
     override fun displayTeiCoordinates(
         teiCoordinatesData: Pair<TrackedEntityType, TrackedEntityInstance>?
     ) {
+        val featureType = teiCoordinatesData!!.first.featureType() ?: FeatureType.NONE
```

Comment:
Could teiCoordinatesData be null sometime?

Reply:
No it could not. `teiCoordinatesData` is a pair of TrackedEntityType and TrackedEntityInstance. Every TrackedEntityInstance will have a TrackedEntityType.